src/user-util/index.ts

```diff
@@ -45,6 +45,87 @@ ${el}`;
   }
 };
 
+// Why is this hardcoded?
+//   So that the snapshots do not fail when a new version is released and all the error messages change
+// Why is this not pointing to `main`?
+//   So that if the docs are moved around or renamed in the future, the links in previous PT versions still work
+// Does this need to be updated before every release?
+//   No, only when the docs are changed
+const docsVersion = 'v2.0.0';
+const customizeDocsMessage = `You can customize this check by setting the targetSize option, more details at https://github.com/cloudfour/pleasantest/blob/${docsVersion}/docs/errors/target-size.md`;
+
+export const assertTargetSize = (
+  el: Element,
+  targetSize: number | true | undefined,
+) => {
+  const { width, height } = el.getBoundingClientRect();
+
+  const display = getComputedStyle(el).display;
+
+  // Per the W3C recommendation, inline elements are excluded
+  // See: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html
+  if (display === 'inline') {
+    return;
+  }
+
+  const size = typeof targetSize === 'number' ? targetSize : 44;
+
+  if (width < size || height < size) {
+    if (el instanceof HTMLInputElement && el.type === 'checkbox') {
```

Comment:
Smart switching to the label size. :+1: Out of curiosity do we have/plan to add more general actionability checks for un-labeled inputs?

Reply:
Should radio buttons get the same treatment?

Reply:
Radio buttons should, yeah 👍 

Other kinds of inputs, I'm not sure when it makes sense to call .click on them, since usually you would call .type on them. I guess you could have an input type="submit" that maybe you would click? But it should probably be large enough to click easily. And for other kinds of (text/number) inputs, I think they should generally be large enough to click easily.

@gerardo-rodriguez I'm curious what you think as well

> do we have/plan to add more general actionability checks for un-labeled inputs?

We will probably add all the checks from here, and decide which ones make sense for which methods and which kinds of elements: https://playwright.dev/docs/actionability/

Reply:
> @gerardo-rodriguez I'm curious what you think as well


I agree that we could also include:
- radio buttons
- input type="submit"

😉

Reply:
Is it common for `input type="submit"` to be labeled? I haven't run into that. If not it seems like it should need to be large enough by itself.

Reply:
> Is it common for input type="submit" to be labeled?

Can you elaborate, @Paul-Hebert, I'm not sure I understand. 🤔

Reply:
@Paul-Hebert Oh, I think I understand now. You were asking if it's common for an `input type="submit"` to have a `<label>`, correct?

I don't know that I've seen that pattern. It seems it'd be the `input type="submit"` by itself, which I agree in this case, it should be large enough on its own.

Reply:
I'm looking into this now. 🔍 👀

Reply:
Yeah, exactly :+1:

I assumed that radio buttons/checkboxes got special treatment because they're tiny by default but can be activated by clicking their label.

Reply:
Yeah I think that input type="submit" is visually represented as a button anyway so requiring it to be large enough (without checking for a label) seems reasonable to me

Reply:
I have refactored this file. 👍